tools/accuracy_checker/accuracy_checker/representation/segmentation_representation.py

```diff
@@ -231,3 +231,50 @@ def remove(self, indexes):
 
     def to_annotation(self, **kwargs):
         return CoCoInstanceSegmentationAnnotation(self.identifier, self.mask, self.labels)
+
+class OAR3DTilingSegmentationAnnotation(SegmentationAnnotation):
+    def __init__(self, identifier, path_to_mask, depth, height, width, wDepth, wHeight, wWidth):
+        super().__init__(identifier, path_to_mask, GTMaskLoader.NUMPY)
+        self.depth = depth
+        self.height = height
+        self.width = width
+        self.wDepth = wDepth
+        self.wHeight = wHeight
+        self.wWidth = wWidth
+        self.metadata['cD'] = self.depth
+        self.metadata['cH'] = self.height
+        self.metadata['cW'] = self.width
+
+    def _load_mask(self):
+        if self._mask is None:
+            loader_config = self.LOADERS.get(self._mask_loader)
+            data_source = self.metadata.get('segmentation_masks_source')
+            if data_source is None:
+                data_source = self.metadata['data_source']
+            if isinstance(loader_config, str):
+                loader = BaseReader.provide(loader_config, data_source)
+            else:
+                loader = BaseReader.provide(loader_config['type'], data_source, config=loader_config)
+            if self._mask_loader == GTMaskLoader.PILLOW:
+                loader.convert_to_rgb = False
+            mask = loader.read(self._mask_path)
+
+            inputs = mask['inputs']
```

Comment:
in majority of previous cases mask is numpy array, not dictionary, so your code will be processed incorrect (failed with error) if you use another reader, which mentioned in this method

Reply:
Variable naming control moved to accuracy test configuration file